recipe/build.sh

```diff
@@ -56,5 +57,7 @@ if [[ "${CONDA_BUILD_CROSS_COMPILATION:-}" == "1" ]]; then
   mv Makefile.temp Makefile
 fi
 
-CPATH=$PREFIX/include make -j$CPU_COUNT
+# just build the parts we need
+CPATH=$PREFIX/include make -j$CPU_COUNT sub-QtWebKit
+CPATH=$PREFIX/include make -j$CPU_COUNT sub-QtWebKitWidgets
```

Comment:
interesting

Reply:
Yeah it's a bit messy but hopefully I can get it to work. I guess if we can get this going here without rocking the the `pyqt` boat that would be best, we understand that `qtwebkit` is seriously un-cool these days but QGIS still needs it (see discussion at https://github.com/conda-forge/qgis-feedstock/pull/242).

Reply:
Yeah, "high touch" recipes are hard to break into. This is great actually, I think a step in the right "direction".